src/main/java/org/jenkinsci/plugins/pipeline/githubstatusnotification/GitHubStatusNotificationStep.java

```diff
@@ -216,6 +219,24 @@ public String getTargetUrl() {
                 CredentialsMatchers.instanceOf(type)));
     }
 
+    /**
+     * Uses proxy if configured on pluginManager/advanced page
+     *
+     * @param host GitHub's hostname to build proxy to
+     *
+     * @return proxy to use it in connector. Should not be null as it can lead to unexpected behaviour
+     */
+    @Nonnull
+    private static Proxy getProxy(@Nonnull String host) {
+        Jenkins jenkins = GitHubWebHook.getJenkinsInstance();
```

Comment:
I am not sure if this is the best way to access the jenkins instance... I understand that the only reason why you introduced `GitHubWebHook` dependency was to get access to the running jenkins instance... Is there any reason for this or is just a way you found to access the jenkins instance? 

If there is no special reason I would recommend  to use `Jenkins.getInstance()`, this way we do not introduce dependencies  and in the end [GitHubWebHook.getJenkinsInstance](https://github.com/jenkinsci/github-plugin/blob/93d40692ff3866705175624e93ec584d4ac88132/src/main/java/com/cloudbees/jenkins/GitHubWebHook.java#L147) is just a little wrapper over `Jenkins.getInstance()`